code/iaas/accounts/src/main/resources/META-INF/cattle/accounts/spring-accounts-context.xml

```diff
@@ -121,6 +121,30 @@
             </list>
         </property>
     </bean>
+    
+    <bean id="Project" class="io.cattle.platform.api.schema.ObjectBasedSubSchemaFactory" >
+        <property name="id" value="project" />
+        <property name="schemaFactory" ref="User" />
+        <property name="postProcessors">
```

Comment:
Post processors are not needed because the schema is already setup from the "User" SchemaFactory